<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pharmacyprojectClass</class>
 <widget class="QMainWindow" name="pharmacyprojectClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>pharmacyproject</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>251</width>
      <height>441</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Medication List</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="adminMedicationList"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>291</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="medicationLabel">
       <property name="text">
        <string>Medication</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="medicationLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="concentrationLabel">
       <property name="text">
        <string>Concentration</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="concentrationLineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="producerLabel">
       <property name="text">
        <string>Producer</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="producerLineEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="stockLabel">
       <property name="text">
        <string>Stock</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="stockLineEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="valabilityDayLabel">
       <property name="text">
        <string>ValabilityDay</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="valabilityDayLineEdit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="valabilityMonthLabel">
       <property name="text">
        <string>ValabilityMonth</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="valabilityMonthLineEdit"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="valabilityYearLabel">
       <property name="text">
        <string>ValabilityYear</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="valabilityYearLineEdit"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="priceLabel">
       <property name="text">
        <string>Price</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="priceLineEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>250</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="removeButton">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addButton">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="updateButton">
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>330</y>
      <width>291</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Search Bar</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="searchLineEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="searhButton">
         <property name="text">
          <string>Search</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="pharmacyproject.qrc"/>
 </resources>
 <connections/>
</ui>
